roles/openshift_master/meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_docker
```

Comment:
I moved openshift_cli back to it's original state of only including openshift_docker when openshift.common.is_containerized  do you think I should always include it there instead?